staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -873,6 +873,13 @@ type KubeletConfiguration struct {
 	// Default: false
 	// +optional
 	FailCgroupV1 *bool `json:"failCgroupV1,omitempty"`
+
+	// SubidsPerPod is the mapping length of subids for UserNS.
+	// The length must be multiple of 65536.
+	// Default: 65536
+	// +featureGate=UserNamespaceSupport
+	// +optional
```

Comment:
I think new API fields need to be a pointer

Reply:
new optional API fields should be a pointer: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required